plugins/woocommerce-admin/client/products/product-validation.ts

```diff
@@ -10,6 +10,11 @@ import {
 } from '@woocommerce/data';
 import type { FormErrors } from '@woocommerce/components';
 
+/**
+ * Internal dependencies
+ */
+import { isValidSlug } from './sections/utils';
```

Comment:
Instead of implementing `isValidSlug` in utils (which feels like a junk drawer), what about implementing it directly in this `product-validation.ts` file and exporting it from here. Feels like a better place for it, and still allows re-use.

Reply:
I was actually thinking this should be the opposite and elevated up to a more global utils place to use since we'll end up using this in more places soon.

Reply:
I'm not a fan of utils, in general. They always feel like a "junk drawer" for functions.